src/app/screens/Articles/components/ArticleItem/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { View, Image } from 'react-native';
+import CustomText from '@components/CustomText';
+import { Article } from '@interfaces/articlesInterface';
+import { formatDate } from '@utils/dateUtils';
+
+import styles from './styles';
+
+interface Props {
+  item: Article;
+}
+
+function ArticleItem({
+  item: {
+    title,
+    description,
+    updatedAt,
+    author: { image, username }
+  }
+}: Props) {
+  return (
+    <View style={styles.container}>
+      <View style={styles.containerImage}>
+        <Image source={image} style={styles.image} />
```

Comment:
don't you have any warning? the example json has urls, you have to use {uri: image} and you would have to add an image in case the url does not exist